tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -1061,7 +1063,39 @@ void test_locale() {
     assert(stream(year_month_weekday_last{2021y / May / Tuesday[last]}) == STR("2021/Mai/Di[last]"));
 }
 
+template <typename T>
+concept ostream_insertable = requires(std::ostream& o, const T& t) { o << t; };
+
+template <typename Dur>
+void check_stream_insertion_operator_for_duration() {
+    if constexpr (ostream_insertable<sys_time<Dur>>) {
+        std::cout << sys_time<Dur>{};
+    }
+    if constexpr (ostream_insertable<local_time<Dur>>) {
+        std::cout << local_time<Dur>{};
+    }
+}
+
+// Test based on example in LWG-4257 (https://cplusplus.github.io/LWG/issue4257)

```

Comment:
We just say LWG-4257 without a URL. We expect readers to know how to look up issues, with [Custom Autolinks](https://github.com/microsoft/STL/wiki/Custom-Autolinks) in the repo or search engine shortcuts as described on that page.